### DISCUSSION TITLE
[tsml_next_meetings count="5"] on mobile runs listings off page

### DISCUSSION THREAD

**Discussion by RockfaceIT** [code4recovery/12-step-meeting-list#1425] [Q&A]
On our homepage we use this shortcode:
[tsml_next_meetings count="5"]

It looks fine on desktop and tablet.  However, on mobile view it runs beyond the window and messes up the page.

![Home - Al-AnonAlateen Ventura County](https://github.com/code4recovery/tsml-ui/assets/37557944/f9ea3888-d300-4eb5-97bb-219069a133e6)

Look here on mobile:
https://alanonventura.org/

Make sure it's during the day when there's an upcoming meeting.

Is there a way to list upcoming meetings but only listing name and time or something to short/reduce the block?

thank you!

**Comment by SIA-WEBSUP** [comment]
I checked this on my iPhone 12 and there is no extra padding.
I also checked on my laptop in responsive mode and it looked fine as well.

Your website appears to be built using Elementor.
The first thing I would try is forcing the padding to be zero within the shortcode widget, since often times there is a default padding of something other than zero. You might doing this just for small screens.

You could also use some custom CSS to force the padding to zero.
The table has a class of` 'tsml__next_meetings'`

You can, if you wish hide individual columns.
Each column has it's unique class:
![image](https://github.com/code4recovery/12-step-meeting-list/assets/63924390/1203ec05-38c9-40ed-ae79-b83a84071c2a)

**Comment by joshreisner** [comment]
maybe try the "Style with CSS?" checkbox? it is designed for a smaller area

<img width="137" alt="Screenshot 2024-05-22 at 7 17 08 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/dd89f7ca-0ef0-4cb4-9964-603e6e6df52f">
